### ISSUE TITLE
Tooltip does not disappear if removed from page

### ISSUE THREAD

**Issue by tim-rue** [vikramlearning/blazorbootstrap#355]
**Describe the bug**
If you remove/hide the tooltip from the page while it's being displayed, it will not disappear until you reload the page.
I noticed this with a button that removes the parent element from the page. The button will disappear but the tooltip stays.
The problem exists for the `<Tooltip>` component and the `<Button TooltipTitle="...">` component. See sample code below.

**To Reproduce**
1. Hover over the button
2. Click the button to remove it
3. Tooltip doesn't disappear

**Expected behavior**
The tooltip should disappear when removed from the page

**Screenshots**
![image](https://github.com/vikramlearning/blazorbootstrap/assets/39702681/fdac8705-2b27-4cb5-a574-7f6a659a63d3)
![image](https://github.com/vikramlearning/blazorbootstrap/assets/39702681/87ab6e21-ea78-4f67-93da-be7c32aad0ad)

**Versions (please complete the following information):**
 - BlazorBootstrap: 1.10.1
 - Blazor Server
 - .NET 7

**Sample code**
```
 @if (displayContent)
    {
        <Tooltip Class="d-inline-block" Title="Remove content" role="button">
            <button class="btn btn-outline-secondary ms-1" @onclick="() => displayContent = false">
                <Icon Name="IconName.Trash"/>
            </button>
        </Tooltip>
        <Button Color="ButtonColor.Secondary"  Outline="true" Class="ms-1" TooltipTitle="Remove content" TooltipPlacement="TooltipPlacement.Top" @onclick="() => displayContent = false">
            <Icon Name="IconName.Trash2"/>
        </Button>
    }


@code {
    bool displayContent = true;
}


```

P.S. Thanks for creating this awesome framework!

**Comment by gvreddy04** [comment]
@tim-rue This is fixed in #327.

**Comment by tim-rue** [comment]
@gvreddy04  Awesome! Also, I noticed that the tooltip doesn't show on hover if the mouse is over an icon inside the button. See the second button of the sample I posted.
